pages/client-libraries/go.mdx

```diff
@@ -1,130 +1,766 @@
 import { Callout } from 'nextra/components'
+import { Steps } from 'nextra/components'
 
-# Go quick start
 
-At the end of this guide, you will have created a simple Go **`Hello, World!`**
-program that connects to the Memgraph database and executes simple queries.
+# GO client guide 
 
-<Callout type="info"> 
+Learn how to create a Go application that connects to the Memgraph database and executes simple queries. 
 
-Go driver
+Memgraph currently depends on the [Neo4j Golang driver](https://pkg.go.dev/github.com/neo4j/neo4j-go-driver/v5/neo4j#pkg-types) for Go. 
+Memgraph and Neo4j both support Bolt protocol and Cypher queries, which means that same driver can be used to connect to both databases. 
+This is very convenient if there is a need to switch between the two databases.  This guide is based on the driver version v5 and above. Some examples may 
+not be supported in older versions of the driver.
 
-You can find the official Go driver on
-[GitHub](https://github.com/neo4j/neo4j-go-driver).
+## Quickstart 
 
-</Callout>
+The following guide will demonstrate how to start Memgraph, connect to Memgraph, seed the database with data, run simple read and write queries. 
 
-<Callout type="info"> 
+Prerequisite that are necessary and sould be installed in your local enviroment are following: 
 
-Go Object Modeled Graph (OMG)
+ - [Go programming language](https://go.dev/dl/) 
+ - [Docker](https://docs.docker.com/get-docker/)
 
-If you are looking for something similar to the Object Graph Mapper for Go, check out [`gograph`](https://github.com/prahaladd/gograph). This project aims to provide a mechanism to interact with any graph database using a unified and minimalistic API layer for the core operations on a graph database. It is an open-source project not maintained by the Memgraph team.
+This won't be covered in this guide, but you can find more information on how to install them in the links above.
```

Comment:
I don't think it is necessary to point out, i feel like it's obvious?

Reply:
```suggestion
```